Atomic/util.py

```diff
@@ -83,7 +83,10 @@ def subp(cmd, cwd=None):
     return ReturnTuple(proc.returncode, stdout=out, stderr=err)
 
 
-def check_call(cmd, env=os.environ, stdin=None, stderr=None, stdout=None):
+def check_call(cmd, env=os.environ, stdin=None, stderr=None, stdout=None, quiet=True):
+    if quiet:
+        devnull = open(os.devnull, 'w')
+        stdout = devnull
     # Make sure cmd is a list
```

Comment:
I don't see this as an override.  You would keep it to None as the default and only override in the quiet case.